packages/nextra/src/mdx-plugins/static-image.ts

```diff
@@ -82,7 +82,7 @@ export const remarkStaticImage: Plugin<
             name: 'alt',
             value: node.alt
           },
-          {
+          !url.endsWith('.svg') && {
```

Comment:
without this, will get an error 
![image](https://user-images.githubusercontent.com/7361780/189539931-9e382659-8b5c-4280-83f7-838822fd3003.png)

Reply:
Good catch, thanks!